packages/sdk/tests/globals.ts

```diff
@@ -90,15 +90,13 @@ export const publicClientPolygon = createPublicClient({
   }),
 });
 
-export async function sendTestTransaction<
-  TAbi extends Abi | readonly unknown[],
-  TFunctionName extends string = string,
->({
-  network = "mainnet",
-  ...args
-}: SimulateContractParameters<TAbi, TFunctionName> & { network?: "mainnet" | "polygon" }) {
-  const publicClient = network === "polygon" ? publicClientPolygon : publicClientMainnet;
-  const testClient = network === "polygon" ? testClientPolygon : testClientMainnet;
+export type Network = "mainnet" | "polygon";
+
+export async function sendTestTransaction<TAbi extends Abi | readonly unknown[], TFunctionName extends string = string>(
+  args: SimulateContractParameters<TAbi, TFunctionName> & { network: Network },
```

Comment:
Let's remove the `network` prop from `.args` here... So basically keep it as it was before.